### PR TITLE
Update revised-stream-4-language.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
added some words about education

### CODE REVIEW

File: docs/revised-stream-4-language.md [hunk 1]
```diff
@@ -70,4 +70,9 @@ TO DO - need examples.
 
 ### Invest in Education
 
-TO DO
+It is important to not only provide tooling for developers to develop memory-safely, but sharing HOW and WHY to do so is equally of value.  There are several areas where educational materials, training, and job aids should be identified or created to help developers quickly learn about memory safe languages, how to use more memory-safe coding techniques in older languages, as well as the threats and vulnerabilities non-memory-safe coding practices can incur.  We propose to create a reference library and educationcal materials on:
```
Comment (nellshamrell) [inline docs/revised-stream-4-language.md:73]:
```suggestion
It is important to not only provide tooling for developers to develop memory-safely, but sharing HOW and WHY to do so is equally of value.  There are several areas where educational materials, training, and job aids should be identified or created to help developers quickly learn about memory safe languages, how to use more memory-safe coding techniques in older languages, as well as the threats and vulnerabilities non-memory-safe coding practices can incur.  We propose to create a reference library and educational materials on:
```

File: docs/revised-stream-4-language.md [hunk 2]
```diff
@@ -70,4 +70,9 @@ TO DO - need examples.
...
+- the threats and vulnerabilities that exist around memory safety coding flaws
+- the benefits of developing using memory safe languages
+- references to authoritative guides for memory-safe languages
```
Comment (nellshamrell) [inline docs/revised-stream-4-language.md:76]: We should standardize on either "memory safe" or "memory-safe" - I am fine with either.

File: docs/revised-stream-4-language.md [hunk 3]
```diff
@@ -70,4 +70,9 @@ TO DO - need examples.
...
+-  educational training material for older languages, such as C & C++, on how to develop with using memory-safety technigues and safe memory handling practices
```
Comment (nellshamrell) [inline docs/revised-stream-4-language.md:77]:
```suggestion
-  educational training material for older languages, such as C & C++, on how to develop using memory-safety techniques and safe memory handling practices
```

File: docs/revised-stream-4-language.md [hunk 4]
```diff
@@ -70,4 +70,9 @@ TO DO - need examples.
...
+-  through blogs, podcasts, conference presentations, and training classes, spread education awareness and skills in the area of memory safe development languages and secure coding techniques
```
Comment (nellshamrell) [inline docs/revised-stream-4-language.md:78]:
```suggestion
-  spread awareness and help others develop skills in memory safe languages and secure coding techniques through blogs, podcasts, conference presentations, and training classes
```

**Review by nellshamrell — CHANGES_REQUESTED** [review]
A few minor suggestions, ty so much for putting this together!

File: docs/revised-stream-4-language.md (hunk 2 above)
Comment (SecurityCRob) [reply docs/revised-stream-4-language.md:76]: great point, agreed.  maybe we hash that out in the call today?

Comment (nellshamrell) [reply docs/revised-stream-4-language.md:76]: In the call we agreed to standardize on:
* "memory-safe"
* "memory safety"
* NOT "memory-safely"
* NOT "unsafety"

**Comment by nellshamrell** [comment]
@SecurityCRob when you have a moment, do you mind correcting the markdown errors?

File: docs/revised-stream-4-language.md (hunk 2 above)
Comment (SecurityCRob) [reply docs/revised-stream-4-language.md:76]: added dashes where appropriate

**Review by SecurityCRob — COMMENTED** [review]
i *think* i did the needful and fixed the missing spaces around a list

**Review by nellshamrell — APPROVED** [review]

**Comment by 2292gogo** [comment]
Herkül

**Comment by 2292gogo** [comment]
Gogo

**Comment by 2292gogo** [comment]
``

**Comment by 2292gogo** [comment]
1. __

**Pull request merged**